Makefile

```diff
@@ -29,7 +29,8 @@ clone: check-env
 		if [ ! -e ${SDX_HOME}/${PREFIX}$${r} ]; then \
 			git clone git@github.com:ONSdigital/${PREFIX}$${r}.git ${SDX_HOME}/${PREFIX}$${r}; \
 		else \
-			echo "  - already exists: skipping"; \
+			echo "Repo already cloned - pulling"; \
```

Comment:
Could this actually be a `make update` rather than part of the `make` always? You may not want to pull on every build for reasons of speed or to avoid conflicts that you don't necessarily want to deal with yet.